lib/Map/TableStructure.js

```diff
@@ -394,6 +394,7 @@ TableStructure.fromCsv = function(csvString, result) {
     csvString += "\r\n";
   }
   var json = csv.toArrays(csvString, {
+    delimiter: "|",
```

Comment:
Hi @davidedelerma 

We need to be careful about hardcoding values that might potentially break parsing of other csv's. 

My preference would be to see this added as a configurable option on the `CsvCatalogItem` which get's passed down to here. And the default should be the current behaviour, rather than the `|`.

Hope that makes sense